src/frontend/src/common/components/library/data_subview/components/TreeCreateHelpLink/index.tsx

```diff
@@ -8,7 +8,7 @@ const TreeCreateHelpLink = (): JSX.Element => {
     "https://docs.google.com/document/d/1_iQgwl3hn_pjlZLX-n0alUbbhgSPZvpW_0620Hk_kB4/edit?usp=sharing";
   return (
     <StyledDiv>
-      <StyledLink target="_blank" rel="noopener" href={HREF}>
+      <StyledLink href={HREF}>
```

Comment:
I feel like for clarity, new tab opening `StyledLink`s should get renamed to `StyledNewTabLink`s. When I first thought this change, I thought there was an accident, and it wasn't until I read the matched `style.ts` that I realized that was where the component change-out had occurred.